assembl/static2/js/app/integration/brightMirror/index.jsx

```diff
@@ -1,18 +1,48 @@
 import React, { Component } from 'react';
```

Comment:
add flow, import types and storybook data if needed and define the type for each const